src/EjectionManager.sol

```diff
@@ -90,35 +87,31 @@ contract EjectionManager is IEjectionManager, OwnableUpgradeable{
                         timestamp: block.timestamp,
                         stakeEjected: stakeForEjection
                     }));
-                    broke = true;
+                    ratelimitHit = true;
                     break;
                 }
 
-                //try-catch used to prevent race condition of operator deregistering before ejection
-                try registryCoordinator.ejectOperator(
+                stakeForEjection += operatorStake;
```

Comment:
When the rate limit is hit, will we not update these state variables or eject the last operator who has their stake pushed onto the stakeEjectedForQuorum?

Reply:
due to the `break`*

Reply:
Ahh nvm this looks good.  The variables are memory, and we don't need to eject that operator because we crossed the rate limit